readme.md

```diff
@@ -26,21 +26,21 @@ LaTeX, which is pronounced «Lah-tech» or «Lay-tech» (to rhyme with «blech»
 
 ## Why use LaTeX?
 
-* LaTeX is free, multiplatform.
-* LaTeX is just a text document (which can be opened by any text editor), readily converted to PDF.
+* LaTeX is free and multiplatform.
+* LaTeX is just a text document (which can be opened by any text editor), which can be converted in many formats like PDF, HTML or Word Documents.

```

Comment:
`tex4ht` is a decent program if it's really necessary, but TeX was designed to output to camera-ready formats like DVI (and the more modern PDF). HTML is not a first-class citizen here.